src/declarations.d.ts

```diff
@@ -48,6 +49,9 @@ declare module 'fastify' {
       thumbnails: {
         service: ItemThumbnailService;
       };
+      membserships: {
+        service: ItemMembershipService;
+      };
     };
     members: { service: MemberService };
```

Comment:
I think we used to have the membership service decorated at the top level (and not nested inside items). WDYT?

Reply:
sounds good, I didn't know any rules to determine if it should be nested or not